docs/proposals/extender-readiness-check.md

```diff
@@ -0,0 +1,78 @@
+# Proposal: Scheduler-extender readiness check
+
+Last updated: 08.07.2021
+
+
+## Abstract
+CSI has no way to verify scheduler-extender deployment status.
```

Comment:
Abstract sounds confusing. I'd rename to `CSI Operator must inform user about scheduler extender deployment status`

Reply:
fixed